app/experimenter/experiments/serializers/entities.py

```diff
@@ -49,6 +49,7 @@ class Meta:
 
 
 class ExperimentVariantSerializer(serializers.ModelSerializer):
+
```

Comment:
nit: I generally go with a new line before the first method, but class properties can go directly under the class name line without a newline.  I just looked in the Google Python Style guide and didn't see something specifically about new line between class line and first property, but the rest of our classes follow that so I don't think it's worth making these ones different?

Reply:
gah will fix, I guess i over fixed the whitespacing that vs code removed